OneCommunityGlobal/HighestGoodNetworkApp
# Vijay Anand - Added unit for updatePassword actions

# Description
Added unit tests for the updatePassword Action file.

## Related PRS (if any):
No related PRs.

## Main changes explained:
Created a test suite for updatePassword.js(Action).
Test cases cover the following scenarios:
  - updatePassword:
  - Verifies the correct API endpoint is called with appropriate data.
  - Confirms successful updates return the correct status code.
  - Ensures the GET_ERRORS action is dispatched with the right payload when an error occurs.
  - forcePasswordUpdate:
  - Ensures the API endpoint is correctly called with the provided data.
  - Confirms a successful forced update returns the status code.
  - Validates the error handling logic by dispatching GET_ERRORS on API failures.

## How to test:
1. Check out the current branch.
2. Run npm install if necessary.
3. Execute the following command: npm test updatePasswordActions.test.js
7. Ensure all test cases pass without errors.

## Screenshots or videos of changes:
<img width="784" alt="image" src="https://github.com/user-attachments/assets/ba7812c2-23fc-4715-86aa-73aa6c43d5d8" />

## Note:
Please ignore the filename in the screenshot. I have corrected the spelling mistake in the file name.
